### ISSUE TITLE
HCLTech

### ISSUE THREAD

**Issue by ArcherN9** [simple-icons/simple-icons#10312]
### Brand Name

HCLTech

### Official Resources for Icon and Color

Extensive documentation available.
https://app.brandcloudlines.com/hcltech/visual-identity/logo

**Comment by PeterShaggyNoble** [comment]
@ArcherN9, to clarify: is this a request to _add_ HCLTech in addition to our existing HCL icon now that the former's logo is different from the latter's?

**Comment by ArcherN9** [comment]
Good question Peter; One I foresaw coming when I didn't see _icon update request_ appended to this post.

The older HCL logo is still used for HCLTech's lesser known subsidiaries albeit with somewhat modifications. I may not be aware of all of them but HCL Healthcare and HCL Foundation still use it.

It may be prudent to add HCLTech instead of a replacement? Thoughts?

**Comment by PeterShaggyNoble** [comment]
Yep, no harm in having both 👍🏻 Would you like to have a read through [our contributing guidelines](https://github.com/simple-icons/simple-icons/blob/develop/CONTRIBUTING.md#adding-or-updating-an-icon) and give it a go yourself?

It might also be an idea to add aliases to our existing HCL icon for their other brands that are still using the same logo (assuming they're within our scope), with custom `hex` values where relevant.

**Comment by ArcherN9** [comment]
I reviewed it. It appears some working knowledge of one of the image editing tools is a pre requisite. I have none. ☹️
